app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comments_counter
```

Comment:
- [ ] Great job on your project, you have done a great job so far, but there are still some issues you still need to address, please kindly add the Rspect unit test that describes the Comment, as you can see according to the project requirement it is mandatory that  Any custom methods in the models should have a corresponding unit test.kindly add unit specs for all of your models' methods consequently each model should have a corresponding unit test